rimage/image_file.go

```diff
@@ -22,6 +23,15 @@ import (
 	ut "go.viam.com/rdk/utils"
 )
 
+// RGBABitmapMagicNumber represents the magic number for our custom header
+// for raw RGBA data. See the comments in DecodeImage and EncodeImage for
+// more information.
```

Comment:
Can you put the link or description of the format here too? Maybe just write out the description, because I'm afraid of link rot, and the potential of the medium article to go 404 at some point.